core/src/test/java/nav/statusplattform/core/repositories/SampleData.java

```diff
@@ -166,17 +163,15 @@ public static List<ServiceEntity> getNonEmptyListOfServiceEntityWithUid(int numb
         return result;
     }
 
-
-    static String getRandomFromArray(ArrayList<String> array) {
-        if (array.size() == 0) {
+    public static <T> T getRandomFromArray(ArrayList<T> array) {
+        if (array.isEmpty()) {
             //Hit skal man ikke komme
-            return null;
+            throw new NullPointerException("array is null");
```

Comment:
No, don't propagate a `NPE` here. Throw instead say an InvalidArgument exception of some sort. And its better to be specific about _what_ is null, without necessarily relying on function/method internal variable names.

E.g. something like
> Expected input argument should be non-empty ArrayList

Reply:
Otherwise, I wanted to congratulate on a nicely written Generic function =)

As stated in [point 3. in the other comment](https://github.com/navikt/statusplattform/pull/54#discussion_r1596900636), this is also something I want you to simplify/update all other test code packages/modules/repositories touched w/any previous PRs from this round of refactoring.